### ISSUE TITLE
The percentage of printing progress is incorrectly stated

### ISSUE THREAD

**Issue by klotzma** [iobroker-community-adapters/ioBroker.klipper-moonraker#135]
Hello,

The percentage of printing progress is incorrectly stated.
The range is currently from 0.01% to 1%.
0.1 is 1% and 1.0% is 100%, that would have to be changed.

Thanks and otherwise a great adapter, thanks for the work!
![fehler_adapter_moonraker](https://user-images.githubusercontent.com/45261052/139064162-4ccf6c93-f668-4f7c-b6a6-27f932fdaa6f.png)

**Comment by Brainbug01** [comment]
yes the adapter is great! will the percent display be corrected?
